res/sections/30-Lez14032018.tex

```diff
@@ -85,21 +88,40 @@ \subsection{Top 20 cause di fallimento start-up}
  \item Bad time to start: magari il prodotto è interessante, ma si comincia con
 un brutto tempismo, o magari quando il mercato non è pronto
 \end{enumerate}
-\todo{La lista è da gestire in qualche altra maniera, o anzi sarebbe meglio
-ritrovare l'istogramma nelle slide del prof.}
 
-\todo{Qui ha fatto un'escursione sui motori di ricerca, spiegando i risultati
-organici da quelli sponsorizzati. Ha senso aggiungere sta cosa al documento?}
+\begin{figure}[H]
+\includegraphics[width=8cm]{chartFailureStartup.png}
+\end{figure}
 
-Nessuna delle voci citata, come vediamo, non è legata all'idea.
+Altre informazioni riguardo alle ragioni di fallimento di una startup si possono
+trovare al link: \url{https://www.cbinsights.com/research/startup-failure-reasons-top/}
+Nessuna delle voci citata, come vediamo, è legata all'idea.
+
+Nessuna delle voci citata, come vediamo, è legata all'idea.
```

Comment:
C'è una ripetizione qui della stessa frase che c'è in linea 99

Reply:
Tolta!
Tolta!
:stuck_out_tongue_closed_eyes: